docs/providers/aws/guide/serverless.yml.md

```diff
@@ -16,115 +16,272 @@ layout: Doc
 
 Here is a list of all available properties in `serverless.yml` when the provider is set to `aws`.
 
+## Root properties
+
 ```yml
 # serverless.yml
 
-service: myService
+# Service name
+service: myservice
 
+# Framework version constraint: '3', '>=2.33', '^2.33'
```

Comment:
This could be useful especially with v3 when you know your service is compatible with v2 and v3, yet you don't want to force everyone to use either v2 or v3 (e.g. in a team where devs have different versions installed).

Reply:
> This could be useful especially with v3 when you know your service is compatible with v2 and v3

That is best expressed with `2 || 3` (so 4, 5 etc are not recognized as supported)

Reply:
Right, I understand that `^2.67 || 3` is technically better, but it's much simpler and obvious for users to go with `>=2.67`, and it's not that risky since major versions are so far apart in time. The thing is that not everyone is familiar with semver range constraints.

Reply:
> it's not that risky since major versions are so far apart in time.

It's risky, and just that is a valid concern imo. It's unlikely for users to revisit the range, then, while many services are destined to be in service for years simply in maintenance mode.

For same reason, you never put such a range in packages published on npm. We're giving poor's man advice that way.

Also I don't understand the argument that `^2.67 || 3` is more "hard" to use. It looks very clean and straightforward to me. 

>  The thing is that not everyone is familiar with semver range constraints.

It's exactly why, we should provide a future-proof examples, that will prevent users running into issues, long term.